tests/integration/conftest.py

```diff
@@ -70,10 +71,8 @@ async def ip_address_list(ops_test: OpsTest, app: Application):
     """
     # Reduce the update_status frequency until the cluster is deployed
     async with ops_test.fast_forward():
-        status_message = app.units[0].workload_status_message  # type: ignore[attr-defined]
-        await ops_test.model.block_until(
-            lambda: "Ingress IP(s)" in status_message, timeout=15 * 60
-        )
+        await ops_test.model.wait_for_idle(status="active")
+    status_message = app.units[0].workload_status_message  # type: ignore[attr-defined]
```

Comment:
For the `type: ignore` this standard asks for them to be documented with a reason why it is reasonable to ignore them: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#static-code-analysis. Is that applicable here?